packages/app/src/systems/Home/hooks/useHome.tsx

```diff
@@ -0,0 +1,22 @@
+import { useMachine, useSelector } from "@xstate/react";
+
+import type { HomeMachineState } from "../machines/homeMachine";
+import { homeMachine } from "../machines/homeMachine";
+
+const selectors = {
+  context: (state: HomeMachineState) => state.context,
+};
+
+export function useHome() {
+  const [state, , service] = useMachine(homeMachine);
```

Comment:
Idk, but IMHO this shouldn't be an `useHome` hook in order to get accounts and balances. Since the account will be something we will need to use globally makes more sense we have something like `useAccount` and use it around all the pages. By having an `useHome` to fetch account and balances, them in another screens you will need to have another `useSomething` that will make the same things, you know? 

You can move this `useHome` and `homeMachine` to an `useAccount` and `accountMachine`

Reply:
And another point is I think we should use `react-query` to just make simple async operations in our database, it handles better with cache in this case and will avoid a bunch of LOC's. Maybe let XState for situations when we have medium to complex state cases. Idk, what do you think @luizstacio ?